node/core/backing/src/lib.rs

```diff
@@ -385,19 +388,26 @@ async fn validate_and_make_available(
 		pov,
 		validator_index,
 		n_validators,
+		span,
 		make_command,
 	} = params;
 
 	let pov = match pov {
 		Some(pov) => pov,
-		None => request_pov_from_distribution(
-			&mut tx_from,
-			relay_parent,
-			candidate.descriptor.clone(),
-		).await?,
+		None => {
+			let _span = span.child("request PoV");
```

Comment:
This is very OCD, but for me spans should all be `named-like-that` :no_mouth:

Reply:
sounds good

Reply:
if we're going this route, please also update the spans

Reply:
that's beyond the scope of this PR